src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main>
+      <div class="calendar">
+        <div class="day day--tue"></div>
```

Comment:
```suggestion
      <div class="calendar  calendar-start-day--tue">
        <div class="day"></div>
```
Цей модифікатор має бути на контейнері, а не на клітинці